donate/payments/sqs.py

```diff
@@ -18,14 +24,40 @@ def sqs_client():
         )
 
 
+def attempt_send_to_sqs(client, payload):
```

Comment:
Similar to my trouble in https://github.com/mozilla/donate-wagtail/pull/1654/files#r861483363, I moved the actual "attempt" to a seperate function, and am sending the items the call needs through **call_args**.